xwiki-platform-core/xwiki-platform-url/xwiki-platform-url-default/src/test/java/org/xwiki/url/internal/DefaultURLSecurityManagerTest.java

```diff
@@ -464,5 +464,31 @@ void parseToSafeURI() throws URISyntaxException
 
         location = "foo://www.x wiki.org/";
         assertParseToSafeThrowSecurity(location, "foo://www.x%20wiki.org/");
+
+        location = "/xwiki/bin/edit/GroupeInformatique/Comptes%20Rendus/03%20f%C3%A9vrier%202023";
```

Comment:
Maybe use something more common than this very french example :)

Reply:
Well French is handy to have plenty of accents :) but ok